Dockerfile

```diff
@@ -1,8 +1,9 @@
 FROM        quay.io/prometheus/busybox:latest
-MAINTAINER  The Prometheus Authors <prometheus-developers@googlegroups.com>
+LABEL org.opencontainers.image.authors="The Prometheus Authors <prometheus-developers@googlegroups.com>"
```

Comment:
Changes like this should be proposed up in the Prometheus repository first and then propagated everywhere else.

Reply:
Thanks. I see in https://github.com/prometheus/prometheus/blob/master/Dockerfile#L2 it's already been changed to:

```docker
LABEL maintainer "The Prometheus Authors <prometheus-developers@googlegroups.com>"
```

I'll change this to match, and suggest using the OCI annotations as a separate issue.